ui/apps/platform/src/Containers/Clusters/DelegateScanning/Components/DelegatedRegistriesList.tsx

```diff
@@ -1,27 +1,71 @@
+// TODO: remove lint override after @typescript-eslint deps can be resolved to ^5.2.x
+/* eslint-disable react/prop-types */
 import React from 'react';
 import {
     Bullseye,
     Button,
     Card,
-    CardBody,
+    CardFooter,
     EmptyState,
     EmptyStateBody,
     Title,
 } from '@patternfly/react-core';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
 
-import { DelegatedRegistry } from 'services/DelegatedRegistryConfigService';
+import {
+    DelegatedRegistry,
+    DelegatedRegistryCluster,
+} from 'services/DelegatedRegistryConfigService';
+import DelegatedRegistriesTable from './DelegatedRegistriesTable';
 
 type DelegatedRegistriesListProps = {
     registries: DelegatedRegistry[];
+    clusters: DelegatedRegistryCluster[];
+    selectedClusterId: string;
+    addRegistryRow: () => void;
+    deleteRow: (number) => void;
+    handlePathChange: (number, string) => void;
+    handleClusterChange: (number, string) => void;
+    // TODO: re-enable next type after @typescript-eslint deps can be resolved to ^5.2.x
+    // updateRegistriesOrder: (DelegatedRegistry[]) => void;
 };
 
-function DelegatedRegistriesList({ registries }: DelegatedRegistriesListProps) {
+function DelegatedRegistriesList({
+    registries,
+    clusters,
+    selectedClusterId,
+    handlePathChange,
+    handleClusterChange,
+    addRegistryRow,
+    deleteRow,
+    // TODO: remove lint override after @typescript-eslint deps can be resolved to ^5.2.x
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    updateRegistriesOrder,
+}: DelegatedRegistriesListProps) {
     return (
         <Card className="pf-u-mb-lg">
             {registries.length > 0 ? (
-                <CardBody>
-                    <p>(table goes here)</p>
-                </CardBody>
+                <>
+                    <DelegatedRegistriesTable
+                        registries={registries}
+                        clusters={clusters}
+                        selectedClusterId={selectedClusterId}
+                        handlePathChange={handlePathChange}
+                        handleClusterChange={handleClusterChange}
+                        deleteRow={deleteRow}
+                        // TODO: remove lint override after @typescript-eslint deps can be resolved to ^5.2.x
+                        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                        // @ts-ignore
+                        updateRegistriesOrder={updateRegistriesOrder}
+                        key="delegated-registries-table"
```

Comment:
For my understanding, what is the reason for a `key` prop here?

Reply:
This is a legacy of trying to get the focus to stay on the "path" inputs as you type; a key here may be redundant now that the key on the rows themselves are surrogate artificial IDs.